html2asketch/model/symbolMaster.js

```diff
@@ -14,6 +14,7 @@ class SymbolMaster extends Base {
   }
 
   setId(id) {
+    super.setId(id);
```

Comment:
Can symbolID be different from objectID? If so, I don't think we should mix those up and have one setter for both 🤔Not sure how to work around this w/o breaking the API though.

[Edit] Hm, maybe we should name the setter `setObjectID` inestead of `setId`?

Reply:
I've changed `setId` in `Base.js` to `setObjectID` and left `setId` for symbol ID only for backwards compatibility.